File: make_matrix.py
import random

size = 100

print("int matrix[{}][{}] = ".format(size, size))
print("{")
for i in range(size):
    if i != 0 :
        print(",", end="")
    print("{", end="");
    for j in range(size):
        if j != 0 :
            print(",", end="")
        print(random.randint(1, 100), end = " ")
    print("}")
print("};")

print("const int STRIDE = {};".format(size))

File: make_matrix.c
#include <stdio.h>
#include <stdlib.h>

int main(){
  int size = 100;
   
  printf("int matrix[%d][%d] = {", size, size);
  for (int i = 0 ; i < size; i++){
    if (i != 0) printf(",");
    printf("{");
    
    for (int j = 0 ; j < size; j++){
      if ( j != 0) printf(",");
      printf("%d ", rand());
    }
    
    printf("}");
  }

  printf("};\n");

  printf("const int STRIDE = %d;", size);
}
